### ISSUE TITLE
New line in parenthetical causes Unexpected Identifier error

### ISSUE THREAD

**Issue by KassieJones** [microsoft/TypeScript#33032]
**Search Terms:**
Unexpected Identifier, New, New Line
**Code**
Inside a test we have this code

```ts
describe("Some tests", () => {
  let MockGroupAccessorClass: GroupAccessor;
  beforeEach(() => {
    MockGroupAccessorClass = td.replace("../../../src/accessors/group.accessor")
    groupAccessorConstructorCaptor = td.matchers.captor();
    mockGroupAccessor = td.object(GroupAccessor.prototype);
    td.when(
      new MockGroupAccessorClass(groupAccessorConstructorCaptor.capture())).thenReturn(mockGroupAccessor);
  })
});
```

**Expected behavior:**

We would expect this code to function identically to this code

```ts
describe("Some tests", () => {
  let MockGroupAccessorClass: GroupAccessor;
  beforeEach(() => {
    MockGroupAccessorClass = td.replace("../../../src/accessors/group.accessor")
    groupAccessorConstructorCaptor = td.matchers.captor();
    mockGroupAccessor = td.object(GroupAccessor.prototype);
    td.when(new MockGroupAccessorClass(groupAccessorConstructorCaptor.capture())).thenReturn(mockGroupAccessor);
  })
});
```

**Actual behavior:**

Tslint reports `Cannot use 'new' with an expression whose type lacks a call or construct signature.`; it does not report this when all the td.when code is on one line, as in the second code snippet.  To rid this error, we changed from using `new` to using `constructor`.  Doing this, however, and running this test results in an `Unexpected Identifier` error on that line.  If you add @ts-ignore inside the td.when, it resolves itself.

**Playground Link:** N/A, typescript playground doesn't have the appropriate dependencies (i.e. `describe`). 

**Related Issues:**  seems somewhat similar to https://github.com/microsoft/TypeScript/issues/11167

**Comment by AnyhowStep** [comment]
You can import packages on the playground.

```
import * as react from "react";
```

**Comment by RyanCavanaugh** [comment]
It really looks like you meant to write
```ts
  let MockGroupAccessorClass: typeof GroupAccessor;
//                            ^^^^^^^
  beforeEach(() => {
    MockGroupAccessorClass = td.replace("../../../src/accessors/group.accessor")
    groupAccessorConstructorCaptor = td.matchers.captor();
    mockGroupAccessor = td.object(GroupAccessor.prototype);
```

As for the error going away, we would need a way to reproduce this to advise further.

Also we don't control tslint; did you mean tsc?
